app/views/layouts/_admin_header.html.erb

```diff
@@ -25,6 +25,7 @@
           <%= content_tag(:li, link_to(t(:team, default: 'Team'), admin_stats_path(interval: 7))) if tickets? && current_user.is_admin? %>
           <%= helpcenter_menu if (knowledgebase? || forums?) && current_user.is_agent? %>
           <%= content_tag(:li, link_to(t(:content, default: "Content"), admin_categories_path), class:'kblink') if knowledgebase? && current_user.role == 'editor' %>
+          <%= content_tag(:li, link_to(t(:internal_content, default: "Internal Content"), admin_internal_categories_path), class:'kblink') if knowledgebase? %>
```

Comment:
I also think this should not be conditional not on `knowledgebase?` but perhaps instead based on a settings variable.